policyengine_us/parameters/gov/states/ny/tax/income/supplemental/incremental_benefit/widow.yaml

```diff
@@ -3,35 +3,42 @@ metadata:
   type: single_amount
   threshold_unit: currency-USD
   amount_unit: currency-USD
+  period: year
   label: New York supplemental tax incremental benefit widow amount
   reference:
-    - title: Section 601 Imposition of tax, (d-2) for 2021 and (d-3) for 2022
+    - title: Section 601 Imposition of tax, (d-2) for 2021, (d-3) for 2022, and (d-4) for 2023
```

Comment:
are we modeling its repeal in 2028?

Reply:
@PavelMakarchuk should i file a separate issue for this?

Reply:
yes please, this PR is already very complicated